dask/array/ghost.py

```diff
@@ -27,78 +15,20 @@ def fractional_slice(task, axes):
     >>> fractional_slice(('x', 2.9, 5.1), {0: 2, 1: 3})  # doctest: +SKIP
     (getitem, ('x', 3, 5), (slice(0, 2), slice(-3, None)))
     """
-    rounded = (task[0],) + tuple(int(round(i)) for i in task[1:])
-
-    index = []
-    for i, (t, r) in enumerate(zip(task[1:], rounded[1:])):
-        depth = axes.get(i, 0)
-        if t == r:
-            index.append(slice(None, None, None))
-        elif t < r:
-            index.append(slice(0, depth))
-        elif t > r and depth == 0:
-            index.append(slice(0, 0))
-        else:
-            index.append(slice(-depth, None))
 
-    index = tuple(index)
+    warn('ghost module will be renamed to overlap, '
```

Comment:
Request to change "will be" -> "has been". This should be done throughout.

Also request to be a bit more descriptive, since this may show up deeper in other functions. Perhaps:

```python
warn("the dask.array.ghost module has been renamed to dask.array.overlap, ...")
```